orc8r/cloud/helm/orc8r/charts/certs/values.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Create orchestrator certs.
+create: true
+
+domainName: localhost
+
+bootstrapper:
+  # 1 year = 8760h
+  duration: 8760h
+  renewBefore: 24h
+
+adminOperator:
+  # 10 year = 87600h
+  duration: 87600h
+  renewBefore: 24h
+  customIssuer:
+  pkcs12:
+    password: password
+
+certifier:
+  # 10 year = 87600h
+  duration: 87600h
+  renewBefore: 24h
+
+nms:
+  # 10 year = 87600h
+  duration: 87600h
+  renewBefore: 24h
+
+  # You can use one of the following Let's Encrypt issuers.
+  # letsencrypt-prod-issuer
+  # letsencrypt-staging-issuer
+  # NOTE: this won't work with wildcard domain.
+  customIssuer:
+
+controller:
+  # 10 year = 87600h
+  duration: 87600h
+  renewBefore: 24h
+  customIssuer:
```

Comment:
Let's also add the note here about using the provided letsencrypt issuers

Reply:
Removed letsencrypt issuers as per our discussion.